src/hooks/useWalletENSAvatar.ts

```diff
@@ -14,46 +12,11 @@ export default function useWalletENSAvatar() {
 
   const updateWalletENSAvatars = useCallback(async () => {
     if (!profilesEnabled) return;
-    const walletKeys = Object.keys(wallets);
-    let updatedWallets;
-    for (const key of walletKeys) {
-      const wallet = wallets[key];
-      for (const account of wallet?.addresses) {
-        const ens = walletNames[account.address];
-        if (ens) {
-          const images = await fetchImages(ens);
-          if (images?.avatarUrl) {
-            let avatarChanged = false;
-            const addresses = wallet.addresses.map((acc: RainbowAccount) => {
```

Comment:
this was iterating though all `wallet.addresses` for each `account of wallet?.addresses` producing that the wallet avatars weren't being set correctly, seen by @ibrahimtaveras00 

<img width="356" alt="image" src="https://user-images.githubusercontent.com/12115171/169430479-cd8ecadb-3984-4246-9df7-695c9dc20633.png">